### ISSUE TITLE
Baseline classifcation for subgraphs reranking

### ISSUE THREAD

**Issue by highly0** [s-nlp/kbqa#43]
After #41 

###  Experiment  with different metrics for our ML model:
1. Accuracy 
2. ROC/AUC
3. PR/AUC

### Train simple classification ML algorithm on our dataset and features. Experiment to see which models have the highest performance. [Sk learn models](https://scikit-learn.org/stable/supervised_learning.html) 

1. Logistic Regression
2. Random Forest Classifier
3. Decision Trees
4. XGBoost / LightGBM / CatBoost
